arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/IterableTest.kt

```diff
@@ -179,36 +179,6 @@ class IterableTest : UnitSpec() {
       }
     }
 
-    "traverse Validated stack-safe" {
```

Comment:
We'd need to bring these tests back for `mapAccumulating` for `Iterable`, `Map`, `NonEmptyList`, `Sequence`. Probably not anymore for single value containers.